src/getting-started/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Getting started - Graphite Design System
+---
+
+# Getting started
+
+<p class="intro">Welcome to Graphite! Graphite is Way2Web’s open-source white-label design system for digital products and experiences.</p>
+
+Get started by following the guide for [designers](/getting-started/designers) or [developers](/getting-started/developers).
+
+## Overview
+
+A design system is a collection of pre-built, reusable assets—components, patterns, guidance, and code—that allows its users to build consistent digital experiences faster. By using the pre-built and universal assets of Graphite, the time teams spend designing and building is minimized. Instead of building and re-building basic elements, they can spend that time customizing their products to address specific client use cases.
+
+Basically, it's a big box of UI Lego pieces, both in design assets & coded components, accompanied by standards and documentation. With a guide on why and how to use them.
+
+## Graphite is open source
+
+Graphite is funded and built by [Way2Web](https://www.way2web.nl), which means we build for the company’s business needs, but we’ve made it open source for anyone to use and contribute back to.
+
+It's available under the terms of the [MIT license](https://github.com/Way2Web/graphite-design-system/blob/master/LICENSE).
+
+## Target platform
+
+Graphite is mainly targeted at browser/desktop applications.
+
+We make sure our components are usable on mobile browsers with touch input. But for mobile apps we recommend using [Ionic](https://ionicframework.com/) (with [Capacitor](https://capacitorjs.com/)) instead of Graphite. Because of their [adaptive styling](https://ionicframework.com/docs/theming/platform-styles), Ionic apps look and feel at home on every device following the device guidelines while still offering some customization for branding. There are design kits available for [Sketch](https://ionicsketch.com/) & [Figma](https://ionicfigma.com/).
```

Comment:
FYI: Don't know if we have to explain why to use a other system

Reply:
Yeah, agreed on second thought. I will reword this to something like we don't recommend using it for mobile apps because it doesn't adhere to the iOS Human Interface Guidelines & Android Material Design guidelines.